__init__.py

```diff
@@ -1,5 +1,3 @@
-# Copyright 2017, Mycroft AI Inc.
```

Comment:
Not sure this is valid to remove according to the Apache License

Reply:
there is basically no mycroft code left here, and the attribution is covered by the fact we are a fork and such

if this is a concern then we should include a NOTICE file instead attributing every upstream, maybe create a new automation to generate something like

- http://webmanual.hyundai.com/STD_GEN5W/AVNT2/EU/French/opensourcesoftwarenotice.html

where we just add mycroft-core as a new lib

related reading https://www.linuxfoundation.org/blog/blog/copyright-notices-in-open-source-software-projects